package/network/services/dropbear/Makefile

```diff
@@ -112,13 +112,19 @@ CONFIGURE_ARGS += \
 # disable legacy/unsafe methods and unused functionality:
 # - DO_MOTD
 # - DROPBEAR_CLI_NETCAT
+# - DROPBEAR_DH_GROUP14_SHA1
 # - DROPBEAR_DSS
+# - DROPBEAR_RSA_SHA1
+# - DROPBEAR_SHA1_HMAC
 DB_OPT_COMMON = \
 	!!LOCAL_IDENT,"SSH-2.0-dropbear" \
 	DEFAULT_PATH,"$(TARGET_INIT_PATH)" \
 	DO_MOTD,0 \
 	DROPBEAR_CLI_NETCAT,0 \
+	DROPBEAR_DH_GROUP14_SHA1,0 \
 	DROPBEAR_DSS,0 \
+	DROPBEAR_RSA_SHA1,0 \
+	DROPBEAR_SHA1_HMAC,0 \
```

Comment:
What is the oldest dropbear version which is still able to connect after these changes? 
I would prefer if at least the dropbear shipped with openwrt 1907 is still able to connect. 
OpenSSH allows to manually allow older algorithms if needed, but dropbear does not allow this. 

There was already a discussion about this in #11194.

Reply:
Tested with dropbear 2019.78 from OpenWrt 19.07 - older client can connect to newer server.